### PR TITLE
Add Lovense_Remote Latest

### PR DESCRIPTION
**Pull request by SmolSoftBoi**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/lovense-remote.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "lovense-remote" do
+  version ":latest"
```
Comment (miccal) [inline Casks/lovense-remote.rb:2]: Please chaneg this to:
```
version "1.3.9"
```

File: Casks/lovense-remote.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 "a21b7e5a3f6e68ee7dd940bb6b8bbf75246d4e4392045672cbdf04aeb006d3c0"
+
+  url "https://lovense.com/files/apps/remote/remote.dmg"
```
Comment (miccal) [inline Casks/lovense-remote.rb:5]: Please add the following `appcast`:
```
appcast "https://www.lovense.com/sextoys/download"
```

File: Casks/lovense-remote.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  name "Lovense Remote"
+  desc "For remote use"
```
Comment (miccal) [inline Casks/lovense-remote.rb:7]: Please change this to:
```
desc "Remote for Lovense products"
```

**Comment by miccal** [comment]
I think this should be submitted to the [`homebrew-cask-drivers` repository.](https://github.com/Homebrew/homebrew-cask-drivers)

File: Casks/lovense-remote.rb (hunk 3 above)
Comment (vitorgalvao) [reply Casks/lovense-remote.rb:7]: More specific: `Remote for Lovense sex toys`.

**Comment by vitorgalvao** [comment]
> I think this should be submitted to the [`homebrew-cask-drivers`](https://github.com/Homebrew/homebrew-cask-drivers) repository.

Correct.

**Pull request closed without merging**